tools/hooks/pre-commit

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 ./tools/sigh lint
 ./tools/sigh check
+git diff --name-only --cached --relative | grep '\.kt[s"]\?$' | xargs ktlint --android --relative --verbose .
+if [ $? -ne 0 ]; then exit 1; fi
```

Comment:
Should we add this to .travis and break the build if it's a PR?

Reply:
Good idea. I'll look into seeing if something similar is possible (getting the diff between the PR and master. Otherwise, it's going to fail pretty hardcore.